### PR TITLE
prometheus-kea-exporter: init at 0.4.1

### PR DESCRIPTION
**Pull request by mweinelt**
##### Motivation for this change

This is the Prometheus exporter for the ISC Kea DHCP daemon, I want to monitor DHCP metrics with it.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Ma27** [comment]
Can you please add a module for the exporter?

**Comment by mweinelt** [comment]
Thanks for your patience, I did a thing.

File: pkgs/servers/monitoring/prometheus/kea-exporter.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ lib, python3Packages }:
```
Comment (WilliButz) [inline pkgs/servers/monitoring/prometheus/kea-exporter.nix:1]:
```suggestion
{ lib, python3Packages, nixosTests }:
```

**Review by WilliButz — APPROVED** [review]
The input `nixosTests` is missing for the package but otherwise this looks good to merge :+1:

File: pkgs/servers/monitoring/prometheus/kea-exporter.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/servers/monitoring/prometheus/kea-exporter.nix]: Good catch, thanks!

**Pull request merged**